### PR TITLE
Remove unused views and their translations.

### LINKED ISSUE #4420: Unused views in projects.view.private
AliasList is not hooked in urls and git grep does not find any other reference. If it should go projects/alias_list.html can be removed too.
Same  for edit_alias.

### PR DESCRIPTION
**Pull request by invinciblycool**
Closes #4420 
Removed unused views, their templates and translations.

### CODE REVIEW

**Comment by stsewd** [comment]
I wonder if this was something to do with https://github.com/rtfd/readthedocs.org/pull/450. But what I can tell, that was to make an alias to projects (not versions).

Also, the model used it this one https://github.com/rtfd/readthedocs.org/blob/4f1a5dc07fd9d55d4284fdb22deae735932b2ec9/readthedocs/builds/models.py#L408-L425

Looks like we aren't using that anymore. I think we can remove all related forms and models too.

@ericholscher @agjohnson do you agree or maybe this is used on the .com site?

**Comment by agjohnson** [comment]
I don't think we are going to use alias, no. This is going to heavily conflict with my PR to update all of our translations, so perhaps it would be best to scope this PR as just removal of aliases, and not lump multiple PRs into one.

**Comment by invinciblycool** [comment]
> I don't think we are going to use alias, no. This is going to heavily conflict with my PR to update all of our translations, so perhaps it would be best to scope this PR as just removal of aliases, and not lump multiple PRs into one.
@agjohnson So should I push a commit which reverts back to the old translations to not arise any conflicts?

**Comment by stsewd** [comment]
@invinciblycool yeah, the PR that Anthony mentions was already merged

**Comment by invinciblycool** [comment]
@stsewd Done.

File: readthedocs/oauth/migrations/0009_auto_20180927_1049.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
```
Comment (stsewd) [inline readthedocs/oauth/migrations/0009_auto_20180927_1049.py]: This migration doesn't look related to this PR

**Comment by invinciblycool** [comment]
@stsewd Please check now.

**Comment by stsewd** [comment]
@invinciblycool there is more code to remove, please check my comments ^

**Comment by invinciblycool** [comment]
@stsewd Apart from removing other occurrences, I have removed `alias.pk` from a test at 3bc3566. Not sure if it breaks anything.

File: readthedocs/builds/migrations/0001_initial.py [hunk 2]
```diff
@@ -60,16 +60,6 @@ class Migration(migrations.Migration):
                 'permissions': (('view_version', 'View Version'),),
             },
         ),
-        migrations.CreateModel(
```
Comment (stsewd) [inline readthedocs/builds/migrations/0001_initial.py]: We should add a new migration that removes this, not edit existing ones

**Comment by invinciblycool** [comment]
@stsewd Could you please check this out.

File: readthedocs/builds/forms.py [hunk 3]
```diff
@@ -4,29 +4,11 @@
 from builtins import object
 from django import forms
 
-from readthedocs.builds.models import VersionAlias, Version
+from readthedocs.builds.models import Version
 from readthedocs.projects.models import Project
```
[the remark this answers is not in the document]
Comment (invinciblycool) [reply readthedocs/builds/forms.py]: Done.
RTD is awesome. You will have a good time here.

**Comment by invinciblycool** [comment]
@stsewd Done

**Comment by ericholscher** [comment]
In prod we only have 2 of these:

```

In [2]: VersionAlias.objects.all()
Out[2]: <QuerySet [<VersionAlias: Alias for Django: 1.2 -> 1.2.X>, <VersionAlias: Alias for Django: dev -> latest>]>

```

Guessing it should be safe to delete them.
